pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -316,13 +342,18 @@ func TestNewYamlConfig(t *testing.T) {
 				brand:          DEFAULT_BRAND,
 				docURL:         DEFAULT_DOC_URL,
 				apiServerURL:   mockAPIServer,
+				providers: operatorv1.ConsoleProviders{
+					Statuspage: &operatorv1.StatuspageProvider{
+						PageID: mockStatuspageID,
+					},
+				},
 			},
-			want: finalCMDefaultOKD,
+			want: finalCMProviders,
```

Comment:
So we have 1 test here, called `TestNewYamlConfig`, updating the `want` from `finalCMDefaultOKD` to `finalCMProviders` is confusing, IMHO.

Our table tests should probably have several cases:

```golang
tests := []struct {
     name string
     args args
     want string
}{ 
     {
          name: "Test NewYamlConfig() with defaults",
          args: args{
              ...
              // no StatusPage.io to ensure that a null value doesn't break our funcs
          },
          want: finalDefaultOKD  // should we inline?  
     },{
          name: "Test NewYamlConfig() with Status.io provider",
          args: args{ 
             ...
             // includes a StatusPage.io value and ensure output is as expected.
          },
          want: finalCMProviders  // should we inline? 
     },
}
```
See [route.TestIsAdmitted()](https://github.com/openshift/console-operator/blob/ed50a574a7d78bd562a5da21e72c2bb5dab331b8/pkg/console/subresource/route/route_test.go#L117), I think we have a couple tests that do a good job of testing the boundaries of funcs